### PR TITLE
👷  profiles: add AMD AI-SDK profile

### PR DESCRIPTION
**Pull request by vnepogodin**

### CODE REVIEW

File: profiles/pci/ai_sdk/profiles.toml [hunk 1]
```diff
@@ -19,3 +19,14 @@ packages = 'cuda cudnn nccl python-pytorch-opt-cuda ollama-cuda tensorflow-opt-c
 device_name_pattern = '(AD|GV|TU|GA|GH|GM|GP)\w+'
 #device_ids = '*'
 
+[rocm-ai-sdk]
+desc = 'AMD AI SDK and related tools'
+nonfree = true
```
Comment (ventureoo) [inline profiles/pci/ai_sdk/profiles.toml:24]: Is this really nonfree?

Comment (vnepogodin) [reply profiles/pci/ai_sdk/profiles.toml:24]: no, its free. its more to simplify things on the installer side

**Comment by ptr1337** [comment]
Hi @superm1 ,

Checking the ROCm docs we see following supported, is that correct?

Mi300 --> GC 9.4.3
Mi200 --> GC 9.4.2
MI100 --> GC 9.4.1

W7900 --> 11.0.0
W7800 --> 11.0.0
W6800 --> 10.3.0

7900XTX --> 11.0.0
7900XT   --> 11.0.0
7900GRE --> 11.0.0

But we are wondering, if we put for example as GC 10.3.0, then this will be also allowed for the 6800XT, is this correct?
We **really** want to avoid, providing support for official not supported cards. Is there an other way to identify supported cards correctly?

**Comment by superm1** [comment]
> Checking the ROCm docs we see following supported, is that correct?

One mistake I see: 

> W7800 --> 11.0.0

This is 11.0.3

> But we are wondering, if we put for example as GC 10.3.0, then this will be also allowed for the 6800XT, is this correct?

From graphics driver point of view it doesn't differentiate between the two AFAIK.

**Comment by vnepogodin** [comment]
Alright

**Review by ptr1337 — APPROVED** [review]
LGTM

**Pull request merged**